### ISSUE TITLE
Maybe problem with "display resolution" warning message?  Not sure.

### ISSUE THREAD

**Issue by jeremymatt** [spyder-ide/spyder#12076]
### What steps will reproduce the problem?

This warning message appeared after I resumed my screen.  I also had a warning message saying that the spyder had detected a change in my screen resolution and recommended a restart to ensure better display (sorry, I should have noted that error message more closely).  The "display resolution" error has appeared once before but I didn't think anything of it.  I am not 100% sure, but I think that both times this appeared was:
1. After unlocking my screen
2. While a remote desktop connection was active 

I don't see any know bugs related to screen resolution but sorry if I missed them.

### Traceback
```python-traceback
  File "C:\Users\jmatt\Anaconda3\envs\KerasGPU\lib\site-packages\spyder\app\mainwindow.py", line 1393, in handle_new_screen
    self.show_dpi_change_message)
TypeError: 'method' object is not connected
```

## Versions

* Spyder version: 4.1.1 
* Python version: 3.6.10
* Qt version: 5.12.5
* PyQt5 version: 5.12.3
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.3.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.3.0 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.13.0 (OK)
jedi =0.15.2                   :  0.15.2 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  0.9.2 (OK)
paramiko >=2.4.0               :  2.7.1 (OK)
parso =0.5.2                   :  0.5.2 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=0.25                  :  2.4.4 (OK)
pyls >=0.31.9;<0.32.0          :  0.31.9 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.0 (OK)
qtconsole >=4.6.0              :  4.7.1 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  2.4.4 (OK)
spyder_kernels >=1.9.0;<1.10.0 :  1.9.0 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  19.0.0 (OK)

# Optional:
cython >=0.21                  :  0.29.15 (OK)
matplotlib >=2.0.0             :  3.2.1 (OK)
numpy >=1.7                    :  1.18.1 (OK)
pandas >=0.13.1                :  0.25.3 (OK)
scipy >=0.17.0                 :  1.4.1 (OK)
sympy >=0.7.3                  :  1.5.1 (OK)
```

**Comment by ccordoba12** [comment]
Duplicate of #11903

**Comment by ccordoba12** [comment]
It'll be fixed in our next version (4.1.2), to be released in the next couple of days.
